frontend/src/pages/UserAccount.tsx

```
@@ -0,0 +1,84 @@
+      {/* Sidebar */}
+      <aside className="hidden lg:block w-64 p-6 mt-10">
+        <nav className="space-y-4">
+          <ul className="space-y-2">
+            {menuItems.map((item) => (
+              <li
+                key={item}
+                className={`cursor-pointer text-[15px] ${
+                  activeTab === item ? "text-black dark:text-slate-100 font-semibold" : "text-gray-500 dark:text-gray-400"
+                }`}
+                onClick={() => setActiveTab(item)}
+              >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)